miniapps/navier/navier_mms.cpp

```diff
@@ -8,6 +8,19 @@
 // MFEM is free software; you can redistribute it and/or modify it under the
 // terms of the BSD-3 license. We welcome feedback and contributions, see file
 // CONTRIBUTING.md for details.
+//
+// Navier MMS example
+//
+// A manufactured solution is defined as
+//
+// u = [pi * sin(t) * sin(pi * x)^2 * sin(2 * pi * y),
+//      -(pi * sin(t) * sin(2 * pi * x)) * sin(pi * y)^2].
+//
+// p = cos(pi * x) * sin(t) * sin(pi * y)
+//
+// The solution is used to compute the symbolic forcing term (right hand side),
+// of the equation. Then the numerical is computed and compared to the exact
```

Comment:
```suggestion
// of the equation. Then the numerical solution is computed and compared to the exact
```

Reply:
Commenting line 244 did the job for navier_mms. Should you comment it out in the repo?

However, the following is still active in navier_kovasznay.

==24916== Conditional jump or move depends on uninitialised value(s)
==24916==    at 0x7011624: fmax (s_fmax.S:26)
==24916==    by 0x1E6806: mfem::navier::NavierSolver::ComputeCFL(mfem::ParGridFunction&, double) (in /home/blaz/develop/chk/mfem/build/miniapps/navier/navier_kovasznay)
==24916==    by 0x1E232C: main (in /home/blaz/develop/chk/mfem/build/miniapps/navier/navier_kovasznay)
==24916==
==24916== Syscall param write(buf) points to uninitialised byte(s)
==24916==    at 0x7476154: write (write.c:27)
==24916==    by 0x5186A84: MPIDI_CH3I_Sock_write (sock.c:2777)
==24916==    by 0x518CBD6: MPIDI_CH3_iStartMsg (ch3_istartmsg.c:86)
==24916==    by 0x514506E: MPIDI_CH3_EagerContigShortSend (ch3u_eager.c:264)
==24916==    by 0x5163D22: MPID_Send (mpid_send.c:129)
==24916==    by 0x505FD81: MPIC_Send (helper_fns.c:142)
==24916==    by 0x4FEB905: MPIR_Reduce_intra_binomial (reduce_intra_binomial.c:150)
==24916==    by 0x4E958FB: MPIR_Reduce_intra_auto (reduce.c:222)
==24916==    by 0x4E95B4A: MPIR_Reduce_impl (reduce.c:291)
==24916==    by 0x4E95515: MPID_Reduce (mpid_coll.h:222)
==24916==    by 0x4E95CB2: MPIR_Reduce (reduce.c:334)
==24916==    by 0x4FCAB28: MPIR_Allreduce_intra_smp (allreduce_intra_smp.c:41)
==24916==  Address 0x1ffeffe448 is on thread 1's stack
==24916==  in frame #3, created by MPIDI_CH3_EagerContigShortSend (ch3u_eager.c:225)